application/helpers/update/updates/Update_489.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace LimeSurvey\Helpers\Update;
+
+/**
+ * Some of it is a copy of update 453 which has been skipped on some installations
+ */
+class Update_489 extends DatabaseUpdateBase
+{
+    public function up()
+    {
+        $columnSchema = $this->db->getSchema()->getTable('{{archived_table_settings}}')->getColumn('attributes');
+        if ($columnSchema === null) {
+            $this->db->createCommand()->addColumn('{{archived_table_settings}}', 'attributes', 'text NULL');
+            $archivedTableSettings = \Yii::app()->db->createCommand("SELECT * FROM {{archived_table_settings}}")->queryAll();
+            foreach ($archivedTableSettings as $archivedTableSetting) {
+                if ($archivedTableSetting['tbl_type'] === 'token') {
```

Comment:
Why not put this condition in the query?

Reply:
That's how it is laid out on the update 450 and 453